### PR TITLE
Update server-manager due to GRPC interface changes

### PR DESCRIPTION
**Pull request by vfusco**
For more information see https://github.com/cartesi/rollups/issues/10

### CODE REVIEW

**Review by gligneul — APPROVED** [review]

File: src/test-server-manager.cpp [hunk 1]
```diff
@@ -954,7 +954,7 @@ static void verify_proof(FinishEpochResponse &response, const Proof &proof, uint
         "Validity proof should have a valid notices epoch root hash");
     ASSERT(validity.has_machine_state_hash() && !validity.machine_state_hash().data().empty(),
         "Validity proof should have a valid machine state hash");
-    ASSERT(validity.keccak_in_hashes_siblings_size() > 0, "Proof should have keccak in hashes siblings");
+    ASSERT(validity.output_hash_in_output_hashes_siblings_size() > 0, "Proof should have keccak in hashes siblings");
```
Comment (diegonehab) [inline src/test-server-manager.cpp]: Fix error message. Are there other instances of the word "keccak" we might have missed?

File: src/test-server-manager.cpp [hunk 2]
```diff
@@ -988,18 +988,18 @@ static void verify_proof(FinishEpochResponse &response, const Proof &proof, uint
             "Received vouchers epoch root hash should match the calculated one");
 
         auto p2 = assemble_merkle_proof(LOG2_ROOT_SIZE, output_hashes_root_hash, output_epoch_root_hash,
-            output_hashes_in_epoch_siblings, validity.input_index());
+            output_hashes_in_epoch_siblings, validity.input_index_within_epoch());
         ASSERT(p2.verify(h),
```
Comment (diegonehab) [inline src/test-server-manager.cpp:992]: Yes there are.

File: src/test-server-manager.cpp [hunk 3]
```diff
@@ -988,18 +988,18 @@ static void verify_proof(FinishEpochResponse &response, const Proof &proof, uint
...
             "OutputValidityProof vouchers_epoch_root_hash and output_hashes_in_epoch_siblings verification failed");
```
Comment (diegonehab) [inline src/test-server-manager.cpp:993]: Yes there are.

File: src/test-server-manager.cpp [hunk 4]
```diff
@@ -968,18 +968,18 @@ static void verify_proof(FinishEpochResponse &response, const Proof &proof, uint
...
+                vouchers_tree.get_node_hash(validity.input_index_within_epoch() << LOG2_KECCAK_SIZE, LOG2_KECCAK_SIZE),
             "Received output hashes root hash should match the calculated one");
 
-        const auto &output_target_hash = get_voucher_keccak_hash(h, validity.input_index());
+        const auto &output_target_hash = get_voucher_keccak_hash(h, validity.input_index_within_epoch());
         auto p1 = assemble_merkle_proof(metadata_log2_size, output_target_hash, output_hashes_root_hash,
-            keccak_in_hashes_siblings, validity.input_index());
+            output_hash_in_output_hashes_siblings, validity.input_index_within_epoch());
         ASSERT(p1.verify(h),
             "OutputValidityProof output_hashes_root_hash and keccak_in_hashes_siblings verification failed");
```
Comment (diegonehab) [inline src/test-server-manager.cpp]: Yes there are.

File: src/test-server-manager.cpp [hunk 5]
```diff
@@ -988,18 +988,18 @@ static void verify_proof(FinishEpochResponse &response, const Proof &proof, uint
...
+                notices_tree.get_node_hash(validity.input_index_within_epoch() << LOG2_KECCAK_SIZE, LOG2_KECCAK_SIZE),
             "Received output hashes root hash should match the calculated one");
 
-        const auto &output_target_hash = get_notice_keccak_hash(h, validity.input_index());
+        const auto &output_target_hash = get_notice_keccak_hash(h, validity.input_index_within_epoch());
         auto p1 = assemble_merkle_proof(metadata_log2_size, output_target_hash, output_hashes_root_hash,
-            keccak_in_hashes_siblings, validity.input_index());
+            output_hash_in_output_hashes_siblings, validity.input_index_within_epoch());
         ASSERT(p1.verify(h),
             "OutputValidityProof output_hashes_root_hash and keccak_in_hashes_siblings verification failed");
```
Comment (diegonehab) [inline src/test-server-manager.cpp]: Yes there are.

**Review by diegonehab — CHANGES_REQUESTED** [review]
Some error messages still refer to old fields.

**Review by diegonehab — APPROVED** [review]

**Pull request merged**